draft-ietf-quic-transport.md

```diff
@@ -785,13 +785,12 @@ flow control limits.
 If a sender runs out of flow control credit, it will be unable to send new data
 and is considered blocked.  A sender SHOULD send a STREAM_DATA_BLOCKED or
 DATA_BLOCKED frame to indicate it has data to write but is blocked by flow
-control limits.  These frames are expected to be sent infrequently in common
-cases, but they are considered useful for debugging and monitoring purposes.
-
-A sender SHOULD NOT send multiple STREAM_DATA_BLOCKED or DATA_BLOCKED frames
-for the same data limit, unless the original frame is determined to be lost.
-Another STREAM_DATA_BLOCKED or DATA_BLOCKED frame can be sent after the data
-limit is increased.
+control limits.  If a sender is blocked for long enough it is possible for the
```

Comment:
Suggested rephrase: "If a sender is blocked for a period longer than the idle timeout ({{name-idle-timeout}}), the connection might get closed even when data is available for transmission. To keep the connection from closing, a sender SHOULD periodically send a STREAM_DATA_BLOCKED or DATA_BLOCKED frame when there are no ack-eliciting packets in flight."